plugins/bm-alternate-overlays

```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/BuzzfeedBlue/bm-alternate-overlays
```

Comment:
```suggestion
repository=https://github.com/BuzzfeedBlue/bm-alternate-overlays.git
```

Please stop closing and opening new prs with the same content. You can review the steps of submitting a plugin [here](https://github.com/runelite/plugin-hub#submitting-a-plugin) if you want.